jdaviz/configs/imviz/plugins/footprints/footprints.py

```diff
@@ -437,6 +438,12 @@ def _on_is_active_changed(self, *args):
                 for mark in marks:
                     mark.visible = visible
 
+        # Notify toolbar to update tool visibility
+        for viewer in self.app._viewer_store.values():

```

Comment:
I think we can be a little cheaper for the multi-viewer case.  Alternatively would it make sense to just put this in the previous for-loop instead of looping again?  

Either way, I think we also need to emit when `viewer_selected` is changed (the for-loop in `_display_args_changed`).

```suggestion
        viewers = self.viewer.selected_obj if self.viewer.is_multiselect else [self.viewer.selected_obj]
        for viewer in viewers:
]
```

Reply:
sorry, I think I made this overly confusing by suggesting two solutions.  Can we have only one loop if we just use the existing mark loop (and forget about checking `viewer.selected_obj`)?  Or will that not give the correct behavior?